test/libsolidity/semanticTests/functionCall/return_size_expectation_mismatch.sol

```diff
@@ -0,0 +1,31 @@
+interface ShortReturn {
+    function f() external pure returns (bytes32);
+}
+contract LongReturn {
+    function f() external pure returns (uint[20] memory) {}
+}
+
+contract Test {
+    function test() public returns (uint) {
+        LongReturn lr = new LongReturn();
+        uint freeMemoryBefore;
+        assembly {
+            freeMemoryBefore := mload(0x40)
+        }
+
+        ShortReturn(address(lr)).f();
```

Comment:
Do we have coverage for the opposite case? I.e. returndata size shorter than expected? If not, please add a test.

Reply:
Added a test case in the latest push.